TombEngine/Scripting/Internal/TEN/Misc/Miscellanous.cpp

```diff
@@ -262,6 +263,18 @@ namespace Misc
 		return std::make_tuple(resX, resY);
 	}
 
+	/// Reset object camera back to Lara and deactivate object camera.
+	//  ObjCamera @tparam camSlotId 0 for Lara
+	//  ObjCamera @tparam Mesh 0 for Laras hip mesh
+	//  ObjCamera @tparam camSlotId 0 for Lara
+	//  ObjCamera @tparam bool false to deactivate object camera
+	static void ResetObjCamera()
+	{
+		int itemNumber1 = 0;
```

Comment:
Why item number here is hardcoded to 0?

Reply:
changed it to LaraItem. Camera should reset back to Lara